src/DataAccessLayer/InMemoryDal.cs

```diff
@@ -217,6 +228,14 @@ public async Task<IEnumerable<string>> GetGenresAsync()
             }).ConfigureAwait(false);
         }
 
+        public async Task<string> GetBenchmarkDataAsync(int size)
+        {
+            return await Task<string>.Factory.StartNew(() =>
+            {
+                return benchmarkData[0..size];
+            }).ConfigureAwait(false);
```

Comment:
*if* we make the size configurable (TInyBench does up to 1GB), we should rewrite this to write 1K batches and then the remainder of bytes. Otherwise we'll have to have a massive memory buffer.